requirements-travis.txt

```diff
@@ -1,2 +0,0 @@
-pep8==1.6.2
-inspektor==0.1.15
```

Comment:
Good, only we could add `requirements.txt` with the optional dependency on `autopep8`, what do you think? (not really necessary, just thinking out loud)